# Marascuilo-Procedure-test 

## Background

The most effective treatments for lysosomal diseases, which are a group of genetic disorders resulting from deficiencies of lysosomal enzymes, should occur before symptoms manifest. And the sooner the treatments begin, the better the results will be. Therefore, early diagnosis is crucial for effective treatments and good clinical outcomes. However, novel mutations are often identified, making the diagnosis and prognosis extremely difficult.

PROVEAN, SIFT, I-MUTANT, PolyPhen, PHD-SNP, PANTHER, SNPs&GO are seven bioinformatics tools for phenotype prediction using machine learning technique, structural information, and evolutionary information. 

## Why I use Marascuilo-Procedure-test?

In order to compare these algorithms to choose a algorithm or a combination of algorithms to achieve a better sensitivity as well as specificity, first we conduct a chi-square contingency table test to compare the behavior of the existing seven algorithms. Then, use the code Marascuilo-Procedure-test to see which algorithm has a significant difference with the best one. After we see which algorithms are the best serveral algorithms, we could look into these algorithms carefully and design a new score to combine these serveral algorithms.

## Results

Previously, we have exploited these in silico tools and established a single amino acid mutation prediction (SAAMP) program, which yields a sensitivity of 94.9% and a specificity of 89.7%. More recently, we developed the SAAMP 3.0© program to optimize the weight pattern and its corresponding criterion so that the greatest sensitivity can be achieved. The sensitivity with the optimal weight pattern and criterion peaks at 98.8%, which outperforms any individual bioinformatics tool. Therefore, SAAMP 3.0© represents the best-in-class algorithm to predict the phenotype of a novel missense mutation, and could provide essential information for early diagnosis and proper treatment allocation.
